executor/runtime/docker/docker.go

```diff
@@ -978,9 +978,10 @@ func (r *DockerRuntime) Prepare(parentCtx context.Context) error { // nolint: go
 				"TITUS_SECCOMP_AGENT_HANDLE_PERF_SYSCALLS": "true",
 			})
 		}
-		if r.c.SeccompAgentEnabledForPerfSyscalls() {
+		if r.c.SeccompAgentEnabledForNetSyscalls() {
 			r.c.SetEnvs(map[string]string{
 				"TITUS_SECCOMP_AGENT_HANDLE_NET_SYSCALLS": "true",
+				"TITUS_SECCOMP_AGENT_TRANSITION_NETNS":    r.c.TaskID() + "trans-netns",
```

Comment:
Do we need this? 
I guess we need to *somehow* tell TSA what to switch into.
Hmm.